smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -148,13 +156,83 @@ def __init__(self, result: bytes) -> None:
 class MachineLearningWorkerCore:
     """Basic functionality of ML worker that is shared across all worker types"""
 
+    @staticmethod
+    def deserialize_message(
+        data_blob: bytes,
+        channel_type: t.Type[CommChannelBase],
+    ) -> InferenceRequest:
+        """Deserialize a message from a byte stream into an InferenceRequest
+        :param data_blob: The byte stream to deserialize"""
```

Comment:
Is ``channel_type`` missing from the doc strings?

Reply:
yes. fixed & added `:returns:`, too